static/css/base.scss

```diff
@@ -356,6 +356,28 @@ $headings-font-family: "Google Sans", "Public Sans", $font-family-sans-serif;
   animation: spin 1s infinite linear;
 }
 
+#spinner-text {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+#spinner-text::after {
+  content: "Still loading ...";
+  z-index: 10;
+  margin-top: 150px;
+  margin-left: 20px;
```

Comment:
is the margin necessary if it's already centered?

Reply:
updated screenshot, can you check?

Reply:
i think it's better to remove since it does look off center now..

Reply:
I tried that but since there is "...", the text part seems off center. The 20px is used to offset that.